README.md

```diff
@@ -88,9 +88,24 @@ make html # create html documentation
 version-mp2-georchestra
 version-mp2
 ```
----
-**NOTE**
 
-When a stable branch is created, kindly update the `version.txt` with the new stable branch name and the same applies to Mapstore2 submodule
-
----
+## Release procedure
+
+All the changes are first delivered as release candidates, turned into release after user acceptance
+
+### Creating release candidate:
```

Comment:
- Given a stable branch in mapstore2-georchestra create a new pre-release and tag for it at 
[Release creation page](https://github.com/georchestra/mapstore2-georchestra/releases/new) using the following naming convention:  `YYYY.MM.xx-RCX-geOrchestra` (eg. 2022.02.00-RC1-geOrchestra), 
where `YYYY.MM` corresponds to the major version and `xx` is the number of the minor release of the same series; `RCX` is the number of release candidate.
- Add an exhaustive description of the pre-release and attach all the needed assets: 
   - *.war file for geOrchestra build
   - Link to the docker image in GeoSolutions Docker Hub
   - Link to the corresponding release page in MapStore as soon as available
   - Relevant configuration steps expected fro the new version
- If a new RC should be released simply keep updated the RC number (eg. 2022.02.00-RC2-geOrchestra)